### PR TITLE
Filenames added to audit log in the LocalIndexUpdated event

### PR DESCRIPTION
**Pull request by nrm21**
Filenames added to audit log in the LocalIndexUpdated event

### CODE REVIEW

File: lib/model/model.go [hunk 1]
```diff
@@ -1127,11 +1127,17 @@ func (m *Model) updateLocals(folder string, fs []protocol.FileInfo) {
 	files := m.folderFiles[folder]
 	m.fmut.RUnlock()
 	files.Update(protocol.LocalDeviceID, fs)
-
+	
+	var filenames []string
```
Comment (calmh) [inline lib/model/model.go]: A little bit more efficient:

```go
filenames := make([]string, len(fs))
for i, file := range fs {
  files[i] = file.Name
```

Comment (nrm21) [inline lib/model/model.go]: did you mean filenames[i] = file.Name ?

**Comment by calmh** [comment]
In principle sound; needs a quick run of the "ManyFiles" benchmark to check the impact of this though, and it needs a rebase.

**Comment by nrm21** [comment]
ok so I'll change the code to reflect what you put...

might be a bit tho... collapsing in bed now (2am)... work tomorrow... sometime over the weekend I'll finish

**Comment by nrm21** [comment]
Nm I did it... I think.

**Comment by calmh** [comment]
@st-jenkins add to whitelist

**Comment by calmh** [comment]
No real impact in the benchmark that I can see.

**Comment by calmh** [comment]
:+1: Thanks

**Comment by nrm21** [comment]
Your welcome.  It's small but hopefully it will help people when looking in the logs for what files changed when.  :)

**Pull request closed without merging**